xml/System.Windows.Xps.Serialization/XpsPackagingPolicy.xml

```diff
@@ -1,862 +1,865 @@
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">reachPackage is NULL.</exception>
+      </Docs>
+    </Member>
+    <Member MemberName=".ctor">
+      <MemberSignature Language="C#" Value="public XpsPackagingPolicy (System.Windows.Xps.Packaging.XpsDocument xpsPackage, System.Windows.Xps.Packaging.PackageInterleavingOrder interleavingType);" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(class System.Windows.Xps.Packaging.XpsDocument xpsPackage, valuetype System.Windows.Xps.Packaging.PackageInterleavingOrder interleavingType) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.Windows.Xps.Serialization.XpsPackagingPolicy.#ctor(System.Windows.Xps.Packaging.XpsDocument,System.Windows.Xps.Packaging.PackageInterleavingOrder)" />
+      <MemberSignature Language="VB.NET" Value="Public Sub New (xpsPackage As XpsDocument, interleavingType As PackageInterleavingOrder)" />
+      <MemberSignature Language="F#" Value="new System.Windows.Xps.Serialization.XpsPackagingPolicy : System.Windows.Xps.Packaging.XpsDocument * System.Windows.Xps.Packaging.PackageInterleavingOrder -&gt; System.Windows.Xps.Serialization.XpsPackagingPolicy" Usage="new System.Windows.Xps.Serialization.XpsPackagingPolicy (xpsPackage, interleavingType)" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; XpsPackagingPolicy(System::Windows::Xps::Packaging::XpsDocument ^ xpsPackage, System::Windows::Xps::Packaging::PackageInterleavingOrder interleavingType);" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>ReachFramework</AssemblyName>
+        <AssemblyVersion>3.0.0.0</AssemblyVersion>
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters>
+        <Parameter Name="xpsPackage" Type="System.Windows.Xps.Packaging.XpsDocument" />
+        <Parameter Name="interleavingType" Type="System.Windows.Xps.Packaging.PackageInterleavingOrder" />
+      </Parameters>
+      <Docs>
+        <param name="xpsPackage">The XML Paper Specification (XPS) document that the packaging policy applies to.</param>
+        <param name="interleavingType">The order in which to interleave document elements.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Windows.Xps.Serialization.XpsPackagingPolicy" /> class with a specified <see cref="T:System.Windows.Xps.Packaging.XpsDocument" /> and specified interleave order.</summary>
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">reachPackage is NULL.</exception>

```

Comment:
```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="xpsPackage"/> is <see langword="null"/>.</exception>

```

Reply:
Is it correct to use `null` or is it actually some object called NULL? It has happened before.